rules/integrations/aws/impact_kms_cmk_disabled_or_scheduled_for_deletion.toml

```
@@ -0,0 +1,73 @@
+[metadata]
+creation_date = "2022/09/21"
+maturity = "production"
+updated_date = "2022/09/21""
+integration = "aws"
+
+[rule]
+author = ["Elastic", "Xavier Pich"]
+description = """
+Detects an attempt to disable or schedule for deletion of an AWS KMS Customer Managed Key (CMK).
+
+Deleting an AWS KMS key is destructive and potentially dangerous. It deletes the key material and all metadata associated with the KMS key and is 
+irreversible. After a KMS key is deleted, you can no longer decrypt the data that was encrypted under that KMS key, which means that data becomes 
+unrecoverable.
+"""
+false_positives = [
+    """
+    A KMS customer managed key may be disabled or scheduled for deletion by a system administrator. Verify whether the user identity, user agent, 
+	and/or hostname should be making changes in your environment. Log stream deletions by unfamiliar users should be investigated. If known 
+	behavior is causing false positives, it can be exempted from the rule.
+    """,
+]
+from = "now-60m"
+index = ["filebeat-*", "logs-aws*"]
+interval = "10m"
+language = "kuery"
+license = "Elastic License v2"
+name = "AWS KMS Customer Managed Key Disabled or Scheduled For Deletion"
+note = """## Setup
+
+The AWS Fleet integration, Filebeat module, or similarly structured data is required to be compatible with this rule."""
+references = [
+    "https://docs.aws.amazon.com/cli/latest/reference/kms/disable-key.html",
+    "https://docs.aws.amazon.com/cli/latest/reference/kms/schedule-key-deletion.html",
+]
+risk_score = 47
+rule_id = "x135f0du-po0z-47c7-2f6f-qsfp1x7cca9n"
```

```suggestion
rule_id = "6951f15e-533c-4a60-8014-a3c3ab851a1b"
```
I don't think this is a valid GUID as it contains non-hex chars